### PR TITLE
SFS Turbo apimon

### PR DESCRIPTION
**Pull request by jmiskani**
Apimon playbook for SFS Turbo

### CODE REVIEW

File: playbooks/scenario46_sfsturbo.yaml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        - block:
+
+           - name: Delete sfs turbo share
+             opentelekomcloud.cloud.sfsturbo_share:
+               name: "{{ test_share_name }}"
+               state: absent
+
+           - name: Pause for 60 seconds
+             pause:
+               seconds: 60
```
Comment (vladimirhasko) [inline playbooks/scenario46_sfsturbo.yaml:111]: please create issue on opentelekomcloud ansible-collection repo to introduce waiting for deletion

**Review by vladimirhasko — CHANGES_REQUESTED** [review]

File: playbooks/scenario46_sfsturbo.yaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+      - name: Create Security group
+        openstack.cloud.security_group:
+          name: "{{ test_security_group_name }}"
+          description: Test security group created by APImon
+
+      - name: Get SecurityGroup info
+        openstack.cloud.security_group_info:
+          name: "{{ test_security_group_name }}"
+        register: security
```
[the remark this answers is not in the document]
Comment (jmiskani) [reply playbooks/scenario46_sfsturbo.yaml:68]: according the documentation:
https://galaxy.ansible.com/ui/repo/published/opentelekomcloud/cloud/content/module/security_group/
there are no return values and to get the sec group ID it needs to be obtained from opentelekomcloud.cloud.security_group_info

File: playbooks/scenario46_sfsturbo.yaml [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+    - block:
+
+      - name: Get Availability zones
+        opentelekomcloud.cloud.availability_zone_info:
+        register: azs
+
+      - name: Set Availability Zone
+        set_fact:
+          az: "{{ azs.availability_zones.0.name }}"
```
[the remark this answers is not in the document]
Comment (jmiskani) [reply playbooks/scenario46_sfsturbo.yaml:30]: updated to randomize the AZ selection

File: playbooks/scenario46_sfsturbo.yaml [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        vars:
+          router_name: "{{ test_router_name }}"
+          network_name: "{{ test_network_name }}"
+          subnet_name: "{{ test_subnet_name }}"
+          state: present
+
+      - name: Get VPC Info
+        opentelekomcloud.cloud.vpc_info:
+          name_or_id: "{{ test_router_name }}"
+        register: vpc_info
```
[the remark this answers is not in the document]
Comment (jmiskani) [reply playbooks/scenario46_sfsturbo.yaml:44]: updated

File: playbooks/scenario46_sfsturbo.yaml (hunk 1 above)
Comment (jmiskani) [reply playbooks/scenario46_sfsturbo.yaml:100]: https://github.com/opentelekomcloud/ansible-collection-cloud/issues/362

File: playbooks/scenario46_sfsturbo.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (vladimirhasko) [reply playbooks/scenario46_sfsturbo.yaml:53]: really?

```
TASK [Create SecurityGroup] ***************************************************************************************************************************************************************************************

    "security_group": {
        "created_at": "2024-11-19T10:48:44",
        "description": "Test Security group created by APImon",
        "id": "14da2ddb-8b07-4ac3-8af5-2f1c96fcac7b",
        "name": "scenario19-ee51602a499c0ad6d023aec261edb2f8-sg",
        "project_id": "****",
        "revision_number": null,
        "security_group_rules": [
            {
                "created_at": "2024-11-19T10:48:44",
                "description": null,
                "direction": "egress",
                "ethertype": "IPv4",
                "id": "519a7054-34e8-4b91-925e-e79dcdb95cbd",
                "port_range_max": null,
                "port_range_min": null,
                "project_id": "***",
                "protocol": null,
                "remote_address_group_id": null,
                "remote_group_id": null,
                "remote_ip_prefix": null,
                "security_group_id": "14da2ddb-8b07-4ac3-8af5-2f1c96fcac7b",
                "tenant_id": "****",
                "updated_at": "2024-11-19T10:48:44"
            },
```

**Review by vladimirhasko — CHANGES_REQUESTED** [review]

File: playbooks/scenario46_sfsturbo.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jmiskani) [reply playbooks/scenario46_sfsturbo.yaml:45]: updated

**Pull request merged**